compiler/rustc_parse/src/lexer/mod.rs

```diff
@@ -656,7 +660,8 @@ impl<'a> StringReader<'a> {
         content_end: BytePos,
         prefix_len: u32,
         postfix_len: u32,
-    ) {
+    ) -> bool {
```

Comment:
Nit: it's unusual that such function returns `true` in the bad case, and `false` in the good case.

Reply:
I will invert it.